arwen/host/execution_vmoutputs_test.go

```diff
@@ -820,6 +833,7 @@ func expectedVMOutputDestCtxByCallerSimpleTransfer(value int64) *vmcommon.VMOutp
 	)
 	childAccount.Balance = big.NewInt(1000)
 	childAccount.BalanceDelta = big.NewInt(-value)
+	childAccount.GasUsed = 666
```

Comment:
this is an interesting number

Reply:
If you simultaneously roll 3 dice, you'll statistically get the combination "6 6 6" once every 216 throws :P